src/application/WorkoutSelectionView.fxml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.collections.*?>
-<?import javafx.scene.text.*?>
-<?import javafx.scene.image.*?>
-<?import javafx.geometry.*?>
-<?import java.lang.*?>
-<?import javafx.scene.control.*?>
-<?import javafx.scene.layout.*?>
-<?import javafx.scene.layout.VBox?>
+<?import java.lang.Integer?>
 <?import javafx.collections.FXCollections?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ChoiceBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Font?>
 
-<VBox fx:id="workoutSelection" alignment="CENTER" prefHeight="200.0" prefWidth="325.0" spacing="10.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="application.WorkoutSelectionViewController">
+<VBox fx:id="workoutSelection" alignment="CENTER" prefHeight="200.0" prefWidth="325.0" spacing="10.0" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="application.WorkoutSelectionViewController">
```

Comment:
Did you make these changes for a reason or was it done automatically?
Notice the change from javafx/8 to javafx/18
The vm uses javafx/8, so I don't know if it will be compatible if we merge this

Reply:
This 18 must be changed back to an 8.
This could cause problems with compatability. Gives me a warning the console:
 WARNING: Loading FXML document with JavaFX API of version 18 by JavaFX runtime of version 16